ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py

```diff
@@ -674,3 +674,32 @@ def test_paddle_rot90(
         k=k,
         axes=tuple(axes),
     )
+
+
+# unique
+@handle_frontend_test(
+    fn_tree="paddle.unique",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        min_dim_size=2,
+        force_int_axis=True,
+        valid_axis=True,
+    ),
+    none_axis=st.booleans(),
+    test_with_out=st.just(False),
+    ground_truth_backend="torch",
```

Comment:
Tests now throw the error `hypothesis.errors.InvalidArgument: test_paddle_unique() got an unexpected keyword argument 'ground_truth_backend'`.